libs/blocks/global-navigation/dark-nav.css

```diff
@@ -1,8 +1,8 @@
 :root {
   --text-color: #f2f2f2;
   --background-color: #111;
-  --link-color: #1473e6;
-  --link-hover-color: #1473e6;
+  --link-color: #5681ff;
```

Comment:
Could you not reference the variables from `styles.css` or can this file be used outside Milo?